src/QMCWaveFunctions/TrialWaveFunction.cpp

```diff
@@ -667,6 +668,38 @@ void TrialWaveFunction::evaluateDerivatives(ParticleSet& P,
   }
 }
 
+void TrialWaveFunction::evaluateDerivativesWF(ParticleSet& P,
+    const opt_variables_type& optvars,
+    std::vector<ValueType>& dlogpsi,
+    bool project)
+{
+  //     // First, zero out derivatives
```

Comment:
Do these commented line still make sense? Update or delete.

Reply:
It's now deleted.